definitions/aircraft/Asobo_DR400.yaml

```diff
@@ -76,4 +75,40 @@ shared:
     var_type: f64
     event_name: LIGHT_POTENTIOMETER_SET
     event_param: 7
-    index_reversed: true
\ No newline at end of file
+    index_reversed: true
+
+  -
+    type: NumSet
+    var_name: A:GENERAL ENG STARTER:1
+    var_units: Bool
+    var_type: i32
+    event_name: SET_STARTER1_HELD
```

Comment:
This can just be an individual event added to electrical.yaml. Everything else is already defined. Setting event value to 1 in tester starts the engine.

Reply:
DA62 also uses the same event. Would work well for all aircraft.